source/Generic/PlayState/PlayState.csproj

```diff
@@ -12,6 +12,8 @@
     <TargetFrameworkVersion>v4.6.2</TargetFrameworkVersion>
     <FileAlignment>512</FileAlignment>
     <Deterministic>true</Deterministic>
+    <NuGetPackageImportStamp>
```

Comment:
What is this?

Reply:
Aarrg, a file that showed the migration from packages.config to ReferencePackage. I removed the file but forgot about the reference here.